tool/jenkins/rat_check.sh

```diff
@@ -22,15 +22,16 @@
 set -exu
 
 workdir=`pwd`
+reponame=incubator-madlib
```

Comment:
That is correct. When the repo moves, this will be addressed here and in the Jenkins projects through MADLIB-1142 (filed yesterday).